crates/bevy_ecs/src/schedule/graph/graph_map.rs

```diff
@@ -408,7 +400,7 @@ mod tests {
     fn node_order_preservation() {
         use NodeId::System;
 
-        let mut graph = Graph::<true>::new();
+        let mut graph = <DiGraph>::default();
```

Comment:
Here's some cursed Rust knowledge:

Yes, `<DiGraph>` and `DiGraph` mean different things here. `<DiGraph>` is equivalent to `<DiGraph<FixedHasher>>` (using the defaults) whereas `DiGraph` is equivalent to `<DiGraph<_>>` (inferring the parameter.

Reply:
Interesting! Is that a deliberate feature of the language or just a quirk in the compiler?